source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -181,36 +190,35 @@ Create a symlink for the network overlays:
 
 .. note::
 
-   |CL| installs CNI plugins that are part of the `cloud-native-basic` bundle to :file:`/usr/libexec/cni`. The directory is required because `swupd verify` uses it, if necessary, to repair a system to a known good state.
-
-.. TODO: Verify whether to omit this section per mythi, eadamsintel.
+   |CL| installs CNI plugins that are part of the `cloud-native-basic`   bundle to :file:`/usr/libexec/cni`. The directory is required because `
+   swupd verify` uses it, if necessary, to repair a system to a known good
+   state.
 
-.. **Notes about Weave Net add-on**
+**Notes about Weave Net add-on**
 
-.. If you choose the `Weave Net` add-on, then you must make the following
-.. changes because it installs itself in the :file:`/opt/cni/bin` directory.
+If you choose the `Weave Net` add-on, then you must make the following
+changes because it installs itself in the :file:`/opt/cni/bin` directory.
```

Comment:
@mythi, please verify for `Weave`. And, please see my comment below regarding correct directory reference.